src/server.ts

```diff
@@ -26,7 +26,7 @@ app.post('/graphql', (req, res) => {
 			operationName: req.body.operationName,
 			variableValues: req.body.variables,
 			contextValue: new Context(
-				'0x19B4ED919081720a753337546284bA7075F21fE6'
+				'0x8c4996794277f6E22F5E2D856Ee7D7782B9c61A8'

```

Comment:
Revert. Don't change this since its not necessary.

Reply:
Doesn't everyone have to change it anyway.

Reply:
Yes, but we don't want to add this into version control for no reason. Why? Let's say Rizwan is working on login and he forked his branch off of a commit that's like 10 versions old. He made his changes and all is good. Now if we decide to put in this here, it will raise a conflict for Rizwan for no reason. This is why, unless it is absolutely necessary, let's not modify files.